lib/datastack/server/ds_class.py

```diff
@@ -56,7 +56,7 @@ def button(self, name, on_click='', args={}):
             click_fn_name =''
 
         block = {
-            "id":self.dynamic_widget_id(),
+            "id":'dsf',
```

Comment:
what is dfs?

Reply:
The args functionality  is not working with dynamic ids. some issue with development environment that we need solve later. to test with fixed id i have put the id as 'dsf'.
we need functionality where user can put its own id and if not then only we generate dynamic id. there is one more commit and pull request in which i have updated the code. so the fixed 'dsf' id will be replaced with new code when we do the merge of both pull request. go through both the pull request and add your review.